data-reconciliation-app/test/utils/jwt-config-generator.ts

```diff
@@ -3,29 +3,29 @@ import * as fs from "fs";
 import * as crypto from "crypto";
 import forge from "node-forge";
 import { KeyPairSyncResult } from "crypto";
-import axios from 'axios';
+import axios from "axios";
 
 /**
  * Create the JWT issuer configs for (Test - Microsoft Azure Identity Provider).
  * This config will be used in sandbox and as proposal for docker and mCCF.
  * Jwt issuer proposal documentation: https://microsoft.github.io/CCF/main/build_apps/auth/jwt.html
-*/
+ */
 export class JwtConfigsGenerator {
-
   public static workspaceFolderPath: string = "./workspace/proposals";
 
-  /*  
+  /*
    * Create JWT issuer proposals for a test Identity Provider for sandbox, docker and mCCF.
    */
   public static async createSandboxTestJwtIssuerConfig(): Promise<any> {
-
     const proposalFilePath = `${this.workspaceFolderPath}/set_jwt_issuer_test_proposal.json`;
     const sandboxConfigFilePath = `${this.workspaceFolderPath}/set_jwt_issuer_test_sandbox.json`;
-    if (fs.existsSync(sandboxConfigFilePath) && fs.existsSync(proposalFilePath)) 
+    if (fs.existsSync(sandboxConfigFilePath) && fs.existsSync(proposalFilePath))
       return;
 
     // make sure the workspace folder exists.
-    await fs.promises.mkdir(this.workspaceFolderPath, { recursive: true }).catch(console.error);
+    await fs.promises
```

Comment:
Your prettier settings printWidth might be set to 80 which makes reading this code a bit hard 🥲.

Would you mind either reversing these formatting changes or setting your prettier to 240 and reformat it so it could look like before 😅?

Reply:
Thanks for pointing it out. I've changed it to 240 :)